dev/diffs/3.4.3.diff

```diff
@@ -2404,7 +2411,31 @@ index 266bb343526..c3e3d155813 100644
        checkAnswer(aggDF, df1.groupBy("j").agg(max("k")))
      }
    }
-@@ -1026,15 +1057,23 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils with Adapti
+@@ -895,6 +928,7 @@ abstract class BucketedReadSuite extends QueryTest with SQLTestUtils with Adapti
+ 
+   test("SPARK-29655 Read bucketed tables obeys spark.sql.shuffle.partitions") {
+     withSQLConf(
++      CometConf.COMET_EXEC_SHUFFLE_WITH_RANGE_PARTITIONING_ENABLED.key -> "false",
```

Comment:
Could you add comments where this is disabled, explaining why we must do this? I know this is needed, but I will likely forget why.